pkgs/development/tools/rust/cargo-valgrind/default.nix

```
@@ -8,18 +8,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-valgrind";
-  version = "1.3.0";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "jfrimmel";
     repo = "cargo-valgrind";
-    # repo lacks the 1.3.0 tag
-    # https://github.com/jfrimmel/cargo-valgrind/issues/33
     rev = "d47dd810e3971d676cde5757df8b2e05ed563e41";
```

The tag should be used instead, e.g.
```suggestion
    rev = version;
```

---

Done; thanks for suggesting it.